### PR TITLE
bump it

### PR DESCRIPTION
**Pull request by AlexBedley**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -948,6 +968,34 @@
         "d2l-typography": "github:BrightspaceUI/typography#semver:^7"
       }
     },
+    "@d2l/d2l-attachment": {
```
Comment (AlexBedley) [inline package-lock.json]: @mdgbayly

File: package-lock.json [hunk 2]
```diff
@@ -3502,12 +3550,13 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#ae70fc0df4835576058a791b407e320a1d7fd520",
+      "version": "github:BrightspaceHypermediaComponents/activities#62c4f075c3b98d39c6e5a0b45231188f223e1058",
```
Comment (AlexBedley) [inline package-lock.json]: @ryantmer @BhamelinD2L

File: package-lock.json [hunk 3]
```diff
@@ -3728,7 +3778,7 @@
       }
     },
     "d2l-enrollments": {
-      "version": "github:BrightspaceHypermediaComponents/enrollments#a56cdf2cf5c936dd9f4f32f0f3e8b1b93c648abb",
+      "version": "github:BrightspaceHypermediaComponents/enrollments#465ae14634d959c9018b37dc2462afb8afb74658",
```
Comment (AlexBedley) [inline package-lock.json]: @BhamelinD2L

File: package-lock.json [hunk 4]
```diff
@@ -4083,7 +4157,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#5db84afccacef096774ad27262e7017c2d478eea",
+      "version": "github:BrightspaceHypermediaComponents/organizations#c8591931aa570385814ee2022264abfe3c0a52bf",
```
Comment (AlexBedley) [inline package-lock.json]: @wongvincent

File: package-lock.json [hunk 5]
```diff
@@ -4108,6 +4182,18 @@
         "siren-parser": "^8.0.0",
         "siren-sdk": "github:BrightspaceHypermediaComponents/siren-sdk#semver:^1",
         "whatwg-fetch": "^3.0.0"
+      },
+      "dependencies": {
+        "siren-sdk": {
```
Comment (AlexBedley) [inline package-lock.json]: @svanherk hey these siren-sdk dependency blocks look new. Is this right?

File: package-lock.json [hunk 6]
```diff
@@ -9193,201 +9294,3657 @@
...
+    "now-and-later": {
+      "version": "2.0.1",
+      "resolved": "https://registry.npmjs.org/now-and-later/-/now-and-later-2.0.1.tgz",
+      "integrity": "sha512-KGvQ0cB70AQfg107Xvs/Fbu+dGmZoTRJp2TaPwcwQm3/7PteUyN2BCgk8KBMPGBUXZdVwyWS8fDCGFygBm19UQ==",
+      "dev": true,
+      "requires": {
+        "once": "^1.3.2"
+      }
+    },
+    "npm": {
```
Comment (AlexBedley) [inline package-lock.json]: woops this is an error

**Comment by AlexBedley** [comment]
i managed to accidentally add npm as a dependency to this which probably isn't right. gonna close this and start again. Sorry for the pings!

File: package-lock.json (hunk 5 above)
Comment (dlockhart) [reply package-lock.json]: FYI Stacey is on vacation

Comment (svanherk) [reply package-lock.json]: Your intuition was right, this is a bad sign - means we have a version mismatch.  Looks like you got it figured out though with https://github.com/BrightspaceHypermediaComponents/activities/pull/517!

**Pull request closed without merging**